e2e/failures/macros.js

```diff
@@ -8,6 +8,6 @@
  */
 'use strict';
 
-module.exports = (one: any, two: any) => {
```

Comment:
this change is breaking CI, why did you make it?

Reply:
Whoops, pushed on accident.